### DISCUSSION TITLE
Lazer freezing then crash after entering map

### DISCUSSION THREAD

**Discussion by Clostron** [ppy/osu#30276] [Q&A]
### Type

Crash to desktop

### Bug description

I am running lazer using the vulkan renderer, my nvidia drivers all up to date. When launching the game everything is fine but as soon as i enter into a map to play it will freeze then crash, I have also had a BSOD but this only happened once. This is happening with both singlethread and multithread.

### Screenshots or videos

![20241014_094718](https://github.com/user-attachments/assets/9eb307e8-3962-48ef-b544-5dc193e47d6e)

### Version

2024.1009.1-lazer

### Logs

![image](https://github.com/user-attachments/assets/cff1855d-289d-4c97-9821-b2cabfeb2a3a)
![image](https://github.com/user-attachments/assets/f548215f-778c-4ab0-babb-1a9608f38c86)
[compressed-logs.zip](https://github.com/user-attachments/files/17361672/compressed-logs.zip)

**Comment by bdach** [comment]
Don't use vulkan if it's broken. It will not receive fixes in its current state. Use any other renderer.

**Reply by Clostron** [reply]
Alright, is there other ongoing issues with vulkan?

**Reply by bdach** [reply]
https://github.com/ppy/osu/discussions?discussions_q=is%3Aopen+vulkan
https://github.com/ppy/osu/issues?q=is%3Aissue+vulkan+is%3Aclosed

**Reply by Clostron** [reply]
Oh, I forgot to search vulkan. Thanks for the help and sorry for wasting time.
